src/traits.rs

```diff
@@ -21,8 +21,28 @@ impl<N> NodeIdT for N where N: Eq + Ord + Clone + Debug + Hash + Send + Sync {}
 pub trait Message: Debug + Send + Sync {}
 impl<M> Message for M where M: Debug + Send + Sync {}
 
-/// Result of one step of the local state machine of a distributed algorithm. Such a result should
-/// be used and never discarded by the client of the algorithm.
+/// Single algorithm step result.
```

Comment:
Did you mean this to be a header? How about
```
## Result of a single algorithm step
```
?

Reply:
No, it's the docstring for `struct Step`. It's just cut off in the diff view =).

Reply:
Still might be worthwhile to normalize this docstring. It's a bit misleading to refer to "result" because `Step` does happen to occur as `Result<Step<..>>` sometimes.
What do you think, @andogro ?

Reply:
Result seems like the most clear word to explain this. If you think it causes confusion for the user because of Result<Step<..>>  would changing 'result' to 'outcome' make it more obvious?

Reply:
I like 'outcome', I'll change it.